pkgs/by-name/on/ondsel/package.nix

```
@@ -0,0 +1,36 @@
+{ appimageTools, lib, fetchurl }:
+
+let
+  pname = "ondsel";
+  version = "2024.1.0";
+  patch = "35694";
+
+  src = fetchurl {
+    # TODO: integrate version rest...
+    url = "https://github.com/Ondsel-Development/FreeCAD/releases/download/${version}/Ondsel_ES_${version}.${patch}-Linux-x86_64.AppImage";
+    hash = "sha256-UNltd0fF2MvVbmANcEaqgEBsrerxqnGCHZXMLGp6OKw=";
+  };
+in
+appimageTools.wrapType2 rec {
+  inherit pname version src;
+
+  extraInstallCommands =
+    let contents = appimageTools.extractType2 { inherit pname version src; };
+    in ''
+      mv $out/bin/${pname}-${version} $out/bin/${pname}
+
+      install -m 444 -D ${contents}/com.ondsel.ES.desktop $out/share/applications/${pname}.desktop
+
+      substituteInPlace $out/share/applications/${pname}.desktop \
+        --replace 'Exec=AppRun' 'Exec=${pname}'
+      cp -r ${contents}/usr/share/icons $out/share
+    '';
+
+
+  meta = with lib; {
+    description = "An engineering suite with an open-source heart (Solo version)";
+    homepage = "https://www.ondsel.com";
+    license = licenses.lgpl2Plus;
+    platforms = [ "x86_64-linux" ];
```

> If this package needs a maintainer to be merged, I could do that.

I think that would be a good idea. Are you in the maintainers list?

```suggestion
    platforms = [ "x86_64-linux" ];
    maintainers = with maintainers; [ b-eyselein ];
    mainProgram = "ondsel";
```

---

I'm not yet in the maintainers list, but I will make a pull request to include myself.

---

Instead of making a new pull request, could you add yourself in a commit above the existing one?

---

Yes, I can